BaikalStandalone/CMakeLists.txt

```diff
@@ -99,6 +99,14 @@ if (WIN32)
     )
 endif ()
 
+if (BAIKAL_ENABLE_DATAGENERATOR)
```

Comment:
Is this required here? Similar code is present in the CMakeLists.txt below.

Reply:
Yes